### PR TITLE
Qna

### PR DESCRIPTION
**Pull request by gulgina84**

### CODE REVIEW

File: client/src/components/questions-answers/individual-question/Index.jsx [hunk 1]
```diff
@@ -1 +1,86 @@
-import React from 'react';
\ No newline at end of file
+import React, { useEffect, useState }from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Container, Row, Col } from 'react-bootstrap';
+import { bindActionCreators } from 'redux';
+import { actionCreators } from '../../../redux/index.js';
+import AnswerList from '/Users/gitayier/Desktop/Catbird/client/src/components/questions-answers/individual-question/AnswerList.jsx';
```
Comment (ChristianStutzman) [inline client/src/components/questions-answers/individual-question/Index.jsx]: Can you change this to a relative location? Can't render that component unless it's relative. Should just be './AnswerList.jsx'

**Review by ChristianStutzman — CHANGES_REQUESTED** [review]
Looks good so far! If you can just change that one file I made a comment on and resubmit we can merge it!

**Comment by gulgina84** [comment]
Fixed! Thanks

**Review by ChristianStutzman — APPROVED** [review]
Perfect! Thank you!

**Pull request merged**